addon/mixins/route.js

```diff
@@ -135,11 +143,21 @@ export default Ember.Mixin.create({
     this.set('_perPage', perPage);
     this.set('_modelPath', modelPath);
     this.set('_extraParams', options);
+    if (typeof boundParams === 'object') {
+      this.set('_boundParams', boundParams);
+    }
 
     var requestPayloadBase = {};
     requestPayloadBase[this.get('perPageParam')] = perPage;
     requestPayloadBase[this.get('pageParam')] = startingPage;
 
+    if (Ember.keys(this.get('_boundParams')).length > 0) {
+      var _this = this;
+      Ember.keys(boundParams).forEach(function(key) {
+        options[key] = _this.get(boundParams[key]);
+      });
+    }
```

Comment:
@hhff sorry I seem to have broken the tests and in my haste forgot to run them. a) is done. b) how? I've tried to privatize and while this works it'll still fail the tests and travis. Appreciate a hint, thanks.